wildfly-builder-image/image.yaml

```diff
@@ -1,20 +1,26 @@
 schema_version: 1
 
-name: &imgName "wildfly/wildfly-centos7"
+name: &imgName "wildfly/wildfly-s2i-jdk11"
 description: "The WildFly s2i image"
 version: &imgVersion "dev"
-from: "centos/s2i-base-centos7"
+from: "registry.access.redhat.com/ubi8/ubi-minimal"
 labels:
     - name: io.k8s.description
-      value: "Platform for building and running JEE applications on WildFly $WILDFLY_VERSION"
+      value: "Platform for building and running JEE applications on WildFly"
```

Comment:
maybe: "Tooling image to build Enterprise Java applications with WildFly"

I'd like to demphasize the "and running" part of the builder image.
And Entreprise Java is more correct (WildFly now does Jakarta EE and MicroProfile)

Reply:
Good suggestion. Fixed.